code/game/gamemodes/blob/powers.dm

```diff
@@ -267,8 +301,107 @@
 	if(!surrounding_turfs.len)
 		return
 
-	for(var/mob/living/simple_animal/hostile/blobspore/BS in living_mob_list)
+	for(var/mob/living/simple_animal/hostile/blob/blobspore/BS in living_mob_list)
 		if(isturf(BS.loc) && get_dist(BS, T) <= 35)
 			BS.LoseTarget()
 			BS.Goto(pick(surrounding_turfs), BS.move_to_delay)
-	return
\ No newline at end of file
+	return
+
+
+/mob/camera/blob/verb/split_consciousness()
+	set category = "Blob"
+	set name = "Split consciousness (100) (One use)"
+	set desc = "Expend resources to attempt to produce another sentient overmind"
+
+	if(!blob_nodes || !blob_nodes.len)
+		src << "<span class='warning'>A node is required to birth your offspring...</span>"
+		return
+	var/obj/effect/blob/node/N = locate(/obj/effect/blob) in blob_nodes
+	if(!N)
+		src << "<span class='warning'>A node is required to birth your offspring...</span>"
+		return
+
+	if(!can_buy(100))
+		return
+
+	verbs -= /mob/camera/blob/verb/split_consciousness //we've used our split_consciousness
+	new /obj/effect/blob/core/ (get_turf(N), 200, null, blob_core.point_rate, "offspring")
+	qdel(N)
+
+	if(ticker && ticker.mode.name == "blob")
+		var/datum/game_mode/blob/BL = ticker.mode
+		BL.blobwincount = initial(BL.blobwincount) * 2
+
+
+/mob/camera/blob/verb/blob_broadcast()
+	set category = "Blob"
+	set name = "Blob Broadcast"
+	set desc = "Speak with your blob spores and blobbernauts as your mouthpieces. This action is free."
+
+	var/speak_text = input(usr, "What would you like to say with your minions?", "Blob Broadcast", null) as text
+
+	if(!speak_text)
+		return
+	else
+		usr << "You broadcast with your minions, <B>[speak_text]</B>"
+	for(var/mob/living/simple_animal/hostile/blob_minion in blob_mobs)
+		blob_minion.say(speak_text)
```

Comment:
whenever blob mobs say something and mob/camera/blob hears it, it runtimes--empty pick from list relevated to hear_say() currently what I'm trying to pin down.